ci-operator/jobs/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.14-periodics.yaml

```diff
@@ -731,3 +731,95 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 0 6 11 2 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: release-v1.14
```

Comment:
We released Openshift Pipelines 1.15.0 recently and the branch to be used is `release-v1.15`

Reply:
@VeereshAradhya - Creating separate [PR#54134](https://github.com/openshift/release/pull/54134/files) for these changes to OpenShift Pipelines v1.15 on OCP 4.17.